gb-emu-lib/CPU.cpp

```diff
@@ -942,65 +944,54 @@ ushort CPU::ReadUShortPC()
 
 void CPU::HandleInterrupts()
 {
-
     // If the IME is enabled, some interrupts are enabled in IE, and
     // an interrupt flag is set, handle the interrupt.
-    if (m_IME)
+    if (m_IME == 0x01)
```

Comment:
I hate C++ for this reason. It's better to check vs a value.  if (somByte) isn't REALLY a boolean statement. It's a hack that C++ lets you do because 0 is FALSE, and !0 isn't.